app/scripts/lib/fxa-client.js

```diff
@@ -112,6 +112,16 @@ define(function (require, exports, module) {
           });
     },
 
+    checkAccountByEmailExists: function (email) {
```

Comment:
I need to pull more context back into my head, for either a signin from signup simplification or dead end on force auth, I need to know both whether the email is registered and whether it has been verified. The case is if an email exists, we send them to signin, which acts like a signup anyways if the email isn't verified.